python/ray/tests/test_failure.py

```diff
@@ -1121,10 +1126,20 @@ def test_node_failure(node_to_kill, use_actors):
         assert wait_for_condition(child_resource_available)
         return node_to_kill
 
-    test_process_failure(use_actors=True)
-    test_process_failure(use_actors=False)
-    node_to_kill = test_node_failure(node_to_kill, use_actors=True)
-    node_to_kill = test_node_failure(node_to_kill, use_actors=False)
+    if node_failure:
+        test_node_failure(node_to_kill, use_actors)
+    else:
+        test_process_failure(use_actors)
+
+    ray.state.state._check_connected()
+    keys = [
```

Comment:
This seems like something we should just be testing in the GCS service instead of needing to reach into redis here

Reply:
Yeah...unfortunately there isn't a good way to do it from Python so I just chose the easiest route here :/